csp-tech-radar.csv

```diff
@@ -8,3 +8,40 @@ Long lived branches with Gitflow,hold,techniques,TRUE,"Gitflow is a strict branc
 Testing as a separate team,hold,techniques,TRUE,"Fast feedback is a core tenet of Agile and critical to the success of a project. Using a separate QA team slows down this feedback, creates an ""us and them"" mentality and makes it more difficult to build quality into the software. Testing should be a tightly integrated activity and isn't something the team can outsource. We recommend integrated teams where testers work closely with developers instead of having testing as a separate organization."
 Anemic REST,hold,techniques,TRUE,"Failure to properly model the domain as a set of resources. Naively developing services that simply expose static, hierarchical data models via templated URLs result in an anemic REST implementation. In a richly modelled domain, REST should enable more than simple repetitive data fetching. In a fully evolved RESTful architecture, business events and abstract concepts are also modelled as resources, and the implementation should make effective use of hypertext, link relations and media types to maximize decoupling between services"
+Spock,adopt,languages & frameworks,FALSE,"A testing and specification framework for Java and Groovy applications. It has been adopted as the standard testing framework for automated functional, integration and SIT tests across the department<br /><br /><p><strong>Teams: </strong>Wrights, Olympus, Krakatoa, Everest, Rockies, Vestrahorn</p>"
+Behave,hold,languages & frameworks,FALSE,"A tool for behavior-driven development (BDD) using tests written in a natural language style, backed up by Python code.<br/>Teams should try to favour BDD tests written with Spock test framework.<br /><br /><p><strong>Teams: </strong>Vesuvius, K2, Rosa</p>"
+Cucumber,hold,languages & frameworks,FALSE,"A software tool that supports behavior-driven development (BDD). Central to the Cucumber BDD approach is its ordinary language parser called Gherkin.<br />Teams should try to favour BDD tests written with Spock test framework.<br /><br /><p><strong>Teams: </strong>Gonzales, K2, Rosa, Fuji</p>"
+Cypress,assess,tools,FALSE,A purely JavaScript-based front-end testing tool.<br />Geb should be preferred instead.<br /><br /><p><strong>Teams: </strong>Gonzales</p>
+Gatling,adopt,tools,FALSE,"An open-source load testing solution for load testing and integrates with development pipelines. <br /><br /><p><strong>Teams: </strong>Wrights, Olympus, Krakatoa, Everest, Rockies, Gonzales, Vestrahorn, K2, Rosa, Fuji</p>"
+Geb,trial,tools,FALSE,"A browser automation solution which can be used for scripting, scraping and general automation. <br /><br /><p><strong>Teams: </strong>Olympus, Rockies, Gonzales, Vestrahorn</p>"
+Hamcrest,trial,languages & frameworks,FALSE,"A framework that assists writing tests in several langs. It supports creating customized assertion matchers, allowing match rules to be defined declaratively.<br /><br /><p><strong>Teams: </strong>Vesuvius, K2, Rosa</p>"
```

Comment:
```suggestion
```
I see Hamcrest as an alternative to Junit assertions, it usually comes as library to be added to the project. As a library I wouldn't add this to the tech radar.